mat-process-api.Tests/V1/Validators/GetProcessRequestValidatorTest.cs

```diff
@@ -0,0 +1,53 @@
+using Bogus;
+using FluentValidation.TestHelper;
+using mat_process_api.V1.Validators;
+using NUnit.Framework;
+
+namespace mat_process_api.Tests.V1.Validators
+{
+
+    [TestFixture]
+    public class GetProcessRequestValidatorTest
+    {
+        GetProcessDocumentRequestValidator validator;
+        private readonly Faker faker = new Faker();
+
+        [SetUp]
+
+        public void SetUp()
+        {
+            validator = new GetProcessDocumentRequestValidator();
+        }
+
+        [Test]
+        [TestCase(null)]
+        [TestCase("")]
+        [TestCase(" ")]
+        public void given_null_or_empty_or_whitespace_processRef_the_validator_should_return_an_error(string processRef)
+        {
+            //assert
+            validator.ShouldHaveValidationErrorFor(req => req.processRef, processRef);
+        }
+
+        [Test]
+        public void given_invalid_guid_string_the_validator_should_return_an_error()
```

Comment:
Need more test cases. Something like [that](https://github.com/LBHackney-IT/mat-process-api/blob/16a76fdc79d75180c9e51c419a93945fd607a0e9/mat-process-api.Tests/V1/Validators/PostProcessImageRequestValidatorTests.cs#L84).